src/Rendering/Images/createImagesRenderingMachine.js

```diff
@@ -148,6 +148,11 @@ function createImagesRenderingMachine(options, context) {
                 to: (c, e) => `imageRenderingActor-${e.data}`,
               }),
             },
+            TOGGLE_LABEL_BBOX: {
+              actions: send((_, e) => e, {
+                to: c => `imageRenderingActor-${c.images.selectedName}`,
```

Comment:
Should this event include the name of the image in the payload like the the other events?  So the UI layer pulls `selectedName` and puts it in `data: { name: selectedName }`.  Then can toss the event name on the `makeTransitions` `forwardToNamedActor` block below. Things are inconsistent, but better every day.

Reply:
Fixed, thanks :+1: